satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -442,6 +465,15 @@ class FakeFCIFileHandlerFDHSI(FakeFCIFileHandlerBase):
                        "grid_type": "2km"},
     }
 
+class FakeFCIFileHandlerFDHSIIQTI(FakeFCIFileHandlerFDHSI):
+    """Mock IQTI for FHDSI data."""
+
+    def _get_test_content_all_channels(self):
+        data = super()._get_test_content_all_channels()
+        data.update({"state/celestial/earth_sun_distance": FakeH5Variable(
+            da.repeat(da.array([np.nan]), 6000), dims=("x"))})
```

Comment:
in the IQT files, is this variable a scalar with one _FillValue or an array? If the former, we should try to keep the test as close to the actual data as possible.

Reply:
@ameraner it is a an array that contains the _FillValue

Reply:
ok, that's fine then, thanks!